sites/kit.svelte.dev/src/lib/Image.svelte

```diff
@@ -0,0 +1,17 @@
+<script>
+	export let src;
+	export let alt;
+	export let width = undefined;
+	export let height = undefined;
```

Comment:
we should add types here

Reply:
I've updated this as well as I can. I'm going to have to expose the `Picture` type from `vite-imagetools` before I can fully do it, but don't want to block on that